src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
     }
```

Comment:
Duplicating lines of code is not a good practice.
Let's take a look:
 
1. We only need a single instance of `Lottery`, because we call its method `getRandomBall()` which will give us a different, random ball everytime. 
2. We can make it more dynamic by creating a constant at the class level `NUMBER_OF_BALLS` and then a loop that uses this constant to specify the amount of iterations.
3. Try using informative variable name for `Lottery` instance. `ball1` suggests its a ball instead of lottery class.